src/main/java/games/strategy/triplea/delegate/Matches.java

```diff
@@ -496,54 +500,71 @@ private Matches() {}
+  static Match<Unit> unitWasUnloadedThisTurn() {
+    return Match.of(obj -> ((TripleAUnit) obj).getUnloadedTo() != null);
+  }
 
-  public static final Match<Unit> UnitWasLoadedThisTurn = Match.of(obj -> ((TripleAUnit) obj).getWasLoadedThisTurn());
+  private static Match<Unit> unitWasLoadedThisTurn() {
+    return Match.of(obj -> ((TripleAUnit) obj).getWasLoadedThisTurn());
+  }
 
-  public static final Match<Unit> UnitWasNotLoadedThisTurn = UnitWasLoadedThisTurn.invert();
+  static Match<Unit> unitWasNotLoadedThisTurn() {
+    return unitWasLoadedThisTurn().invert();
+  }
 
-  public static final Match<Unit> UnitCanTransport =
-      Match.of(unit -> UnitAttachment.get(unit.getType()).getTransportCapacity() != -1);
+  public static Match<Unit> unitCanTransport() {
+    return Match.of(unit -> UnitAttachment.get(unit.getType()).getTransportCapacity() != -1);
+  }
 
-  public static final Match<UnitType> UnitTypeCanProduceUnits =
-      Match.of(obj -> UnitAttachment.get(obj).getCanProduceUnits());
+  public static Match<UnitType> unitTypeCanProduceUnits() {
+    return Match.of(obj -> UnitAttachment.get(obj).getCanProduceUnits());
+  }
 
-  public static final Match<Unit> UnitCanProduceUnits = Match.of(obj -> UnitTypeCanProduceUnits.match(obj.getType()));
+  public static final Match<Unit> UnitCanProduceUnits = Match.of(obj -> unitTypeCanProduceUnits().match(obj.getType()));
 
-  public static final Match<UnitType> UnitTypeHasMaxBuildRestrictions =
-      Match.of(type -> UnitAttachment.get(type).getMaxBuiltPerPlayer() >= 0);
+  public static Match<UnitType> unitTypeHasMaxBuildRestrictions() {
+    return Match.of(type -> UnitAttachment.get(type).getMaxBuiltPerPlayer() >= 0);
+  }
 
-  public static final Match<UnitType> UnitTypeIsRocket = Match.of(obj -> UnitAttachment.get(obj).getIsRocket());
+  public static Match<UnitType> unitTypeIsRocket() {
+    return Match.of(obj -> UnitAttachment.get(obj).getIsRocket());
+  }
 
-  public static final Match<Unit> UnitIsRocket = Match.of(obj -> UnitTypeIsRocket.match(obj.getType()));
+  static Match<Unit> unitIsRocket() {
+    return Match.of(obj -> unitTypeIsRocket().match(obj.getType()));
+  }
 
-  public static final Match<Unit> UnitHasMovementLimit = Match.of(obj -> {
-    final UnitType type = obj.getUnitType();
-    final UnitAttachment ua = UnitAttachment.get(type);
-    return ua.getMovementLimit() != null;
-  });
+  static Match<Unit> unitHasMovementLimit() {
+    return Match.of(obj -> UnitAttachment.get(obj.getUnitType()).getMovementLimit() != null);
+  }
 
-  public static final Match<Unit> UnitHasAttackingLimit = Match.of(obj -> {
-    final UnitType type = obj.getUnitType();
-    final UnitAttachment ua = UnitAttachment.get(type);
-    return ua.getAttackingLimit() != null;
-  });
+  static final Match<Unit> unitHasAttackingLimit() {
+    return Match.of(obj -> UnitAttachment.get(obj.getUnitType()).getAttackingLimit() != null);
+  }
```

Comment:
There was some manual refactoring done here in addition to the field -> method conversion.